group.yml

```diff
@@ -1,4 +1,4 @@
-freeze_automation: false
+freeze_automation: true
```

Comment:
that makes doozer refuse even manual builds. how about:
```suggestion
freeze_automation: scheduled
```

Reply:
til, thanks!